about.html

```diff
@@ -74,21 +116,21 @@ <h2 class="part_title">Partner</h2>
     </section>
     <footer class="footer bg-md-dark">
         <section class="first_f_section">
-        <p class="f_icon">
-            <i class="fa-brands fa-creative-commons"></i>
-        </p>
+        <h2 class="f_icon">
+            <span class="fix_linters_issue">fixing linters issue</span><i class="fa-brands fa-creative-commons"></i>
+        </h2>
         <p class="f_text">
             Companies Tool <br> Good Meeting
         </p>
         </section>
         <section class="second_f_section">
-            Lorem ipsum dolor sit, amet consectetur
+            <h6>Lorem ipsum dolor sit, amet consectetur
             consecteturconsectetur<br>
-            Lorem ipsum dolor sit, amet consectetur
+            Lorem ipsum dolor sit, amet consectetur</h6>
         </section>
     </footer>
```

Comment:
- [ ] The footer has the background color of totally black and it has a logo on the left and text on the right which cover the whole width. Kindly see the design below and make the design similar to it
### Your footer
![yourff](https://user-images.githubusercontent.com/81727988/181705369-c9f2384c-7ec4-495f-88a4-dbd20ca583c6.png)
### Original design
![footer](https://user-images.githubusercontent.com/81727988/181705449-5cbe9482-af83-4ee0-9d86-df83b881fd44.png)

Reply:
Okay thank you, hope I will do it better